cloudfour/cloudfour.com-patterns
# Add PostCSS and SVG inlining

## Overview

Adds PostCSS to our Storybook CSS loader stack, and adds the PostCSS Inline SVG plugin. This will be used in the short-term for embedding a custom `<select>` caret and `<input type="checkbox">` check mark in our CSS with context-appropriate colors.

## Testing

1. Check out branch locally.
1. Install dependencies: `npm ci`
1. Run Storybook: `npm run storybook`
1. Navigate to http://localhost:6006/?path=/docs/components-input--text-elements
1. In your code editor, open `src/components/input/input.scss`
1. Add `background-image: svg-load('icons/caret-down.svg', fill=red);` to the `.c-input` block and save.
1. Observe that the input is now hideously covered in red triangles.

---

- Resolves #582
